sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/AssemblyReference.cs

```diff
@@ -18,62 +18,93 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System;
+using System.Collections.Generic;
```

Comment:
This is the only class who really changed. The rest is just related to using this new class.